cylc/flow/etc/tutorial/runtime-tutorial/flow.cylc

```diff
@@ -1,14 +1,9 @@
-#!jinja2
```

Comment:
This results in 
```
FileParseError: Invalid line (line 104):
   {% from "sys" import path, executable %}
```

Reply:
Might be worth adding a `.validate` script to `cylc/flow/etc/tutorial/runtime-tutorial` to catch this sort of breakage

Reply:
Sorry @MetRonnie  - this PR ought to be draft - I will come back to it ... eventually ...

Reply:
Sorry, I've just got back to this and I think that this is fine - this config is meant to be broken when it is extracted by students. It's not breaking any automated tests is  it?

Reply:
**Note:** These are tutorial resources, they might not be complete working examples. What's important is that they fit with the context of the tutorials.

This PR reverts a previous attempt to "fix" these resources broke the tutorials.

Reply:
The runtime configuration tutorial does not mention adding a Jinja2 shebang. The reason it's needed is because `etc/python-job.settings` contains Jinja2 syntax.

The `.validate` script could just be to run `cylc view` to make sure pre-processing succeeds; it doesn't have to run `cylc validate`

Reply:
@MetRonnie  - sorry - you are absolutely right - I had forgotten that reason I'd taken this out was because I had removed the Jinja2 include (replaced with a Cylc include), and not because the tutorial asks students to re-add it.